.gitignore

```diff
@@ -1,2 +1,3 @@
 /vendor
 /composer.lock
+/phpstan.neon
```

Comment:
What is the purpose of this?

Reply:
Because of phpstan's autodiscovery has this priority: [phpstan.neon > phpstan.neon.dist](https://github.com/phpstan/phpstan/blob/dde08ae/src/Command/CommandHelper.php#L58). Someone who forks this project can have own phpstan.neon, which will not be commited to git repo publicly.

I'm used to use dist files this way, but I can change it for you ofc :-)